### PR TITLE
Rewrite with ES6 syntax

### PR DESCRIPTION
**Pull request by neighborhood999**
Part of important about [prop-types](https://facebook.github.io/react/docs/typechecking-with-proptypes.html).

In some component we use `PropType` from `React`, but in  **v15.5** need using `prop-types` instead.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -127,6 +127,7 @@
     "node-soundcloud": "0.0.5",
     "node-uuid": "^1.4.7",
     "pouchdb": "^3.6.0",
+    "prop-type": "0.0.1",
```
Comment (EragonJ) [inline package.json]: `prop-types` not `prop-type` @@...

Comment (neighborhood999) [inline package.json]: Damn it! Sorry my bad.

Comment (neighborhood999) [inline package.json]: I fixing now, waiting for me.

Comment (EragonJ) [inline package.json]: it's okay, after you finish that, just merge by yourself. Thanks PJ ++

Comment (neighborhood999) [inline package.json]: Sure, EJ are your already review other file?

Comment (neighborhood999) [inline package.json]: I merged it.

**Comment by EragonJ** [comment]
@neighborhood999 thanks!

**Pull request merged**